Sources/Keystore.swift

```diff
@@ -38,7 +38,7 @@ public struct KeyHeader: Codable {
     public var cipherText: String
 
     /// Cipher algorithm.
-    public var cipher: String = "aes-128-ctr"
+    public var cipher: String = "aes-128-cbc"
```

Comment:
Is it possible to support both? I believe keystore v3 could be using either of those

Reply:
Some ref: https://stackoverflow.com/questions/40409876/encrypting-a-private-key-in-ruby-using-aes-128-ctr-scrypt

Reply:
I haven't got to the AES part yet, but I'll look into that.